src/actions/Entities/NameActions.js

```diff
@@ -73,7 +73,7 @@ const select = name => async dispatch => {
     const result = await createPatientVisibility(name);
     if (result) {
       UIDatabase.write(() => {
-        selectedName = createRecord(UIDatabase, 'Patient', name);
+        selectedName = createRecord(UIDatabase, 'Patient', name, true);
```

Comment:
In the same method, but on line 107 there's code:
```
UIDatabase.write(() => createRecord(UIDatabase, 'Patient', patientRecord));
```
Shouldn't we update it with the extra parameter as well? Or maybe we can add a default value for that parameter in `CreateRecord` method?

Reply:
Yes we should update it. And in reselector for some reason default value does not work. If I add a default value, it seem to get set permanently no matter what I sent from outside. I will try some more today to set the default value.

Reply:
Resolved with 564a263c